pallets/bridge/src/lib.rs

```diff
@@ -186,24 +183,17 @@ pub mod pallet {
 	#[pallet::config]
 	pub trait Config:
 		frame_system::Config
+		+ chainbridge::Config
 		+ pallet_balances::Config
 		+ pallet_bridge_mapping::Config
-		+ chainbridge::Config
 		+ pallet_fees::Config
 		+ pallet_nft::Config
 	{
 		/// Specifies the origin check provided by the chainbridge for calls
 		/// that can only be called by the chainbridge pallet.
 		type BridgeOrigin: EnsureOrigin<Self::Origin, Success = Self::AccountId>;
 
-		/// Constant configuration parameter to store the module identifier for the pallet.
-		///
-		/// The module identifier may be of the form ```PalletId(*b"r/bridge")``` (a string of eight characters)
-		/// and set using the [`parameter_types`](https://substrate.dev/docs/en/knowledgebase/runtime/macros#parameter_types)
-		/// macro in one of the runtimes (see runtime folder).
-		#[pallet::constant]
-		type BridgePalletId: Get<PalletId>;
```

Comment:
Lets keep it. We need to fetch the accountID through polkadotjs apps. Else it is not straightforward to know pallet address